src/core/connection.c

```diff
@@ -5101,7 +5130,8 @@ QuicConnRecvPostProcessing(
 
     if (Packet->HasNonProbingFrame &&
         Packet->NewLargestPacketNumber &&
-        !(*Path)->IsActive) {
+        !(*Path)->IsActive &&
+        Connection->Paths[0].Route.RouteState == RouteResolved) {
```

Comment:
Why add this here? What's going to retrigger this path after the resolution completes?

Reply:
This part I have not figured out how to deal with TBH. Any suggestions?